pages/common/7z.md

```diff
@@ -10,6 +10,10 @@
 
 `7z x {{archived}}`
 
+- Extract an archive with user-defined output path
+
+`7z x {{archived}} -o{{output/path}}`
```

Comment:
I suggest let's use `path/to/output` to keep consistency.

Reply:
Good point 👍